webplugin/js/app/kommunicate-ui.js

```diff
@@ -588,7 +588,34 @@ handleAttachmentIconVisibility : function(enableAttachment, msg, groupReloaded)
             startConversationButton.classList.add('force-n-vis');
             hasMultipleConversations ? backButton.classList.remove('force-n-vis') : backButton.classList.add('force-n-vis')
         }
-    }
+    },
+        handleWaitingQueueMessage: function (data) {
+            let groupId = data && data.clientGroupId;
+            let waitingStatus = data && data.metadata.CONVERSATION_STATUS == Kommunicate.conversationHelper.status.WAITING;
+            mckUtils.ajax({
+                type: 'GET',
+                url: MCK_BASE_URL + '/rest/ws/group/waiting/list',
+                global: false,
+                contentType: 'application/json',
+                success: function (res) {
+                    WAITING_QUEUE = res.response;
+                    document.getElementById('mck-waiting-queue-2') && (document.getElementById('mck-waiting-queue-2').innerHTML = "#" + parseInt(WAITING_QUEUE.indexOf(parseInt(groupId)) + 1));
+                    if (waitingStatus) {
+                        kommunicateCommons.modifyClassList({
+                            id: ["mck-waiting-queue"]
+                        }, "vis", "n-vis");
+                    } else {
+                        kommunicateCommons.modifyClassList({
+                            id: ["mck-waiting-queue"]
+                        }, "n-vis", "vis");
+                    }
+
```

Comment:
@Kusum-Dhoundiyal Check the response code is "success" or not if coming.
If nothing is coming to verify if the API was successful or not then you can resolve my comment.